packages/nx-electron/src/executors/build/executor.ts

```diff
@@ -126,6 +126,12 @@ export function executor(
       })
     )
   );
+  for await (const res of results) {
+    if(!res.success) {
+      return res;
+    }
+  }
+  return {success: true};
```

Comment:
Please rename `res` to `result`

Reply:
i think this part does not exist anymore